pkg/controller/issuer/certificate/reconciler.go

```diff
@@ -813,7 +813,9 @@ func (r *certReconciler) copySecretIfNeeded(logctx logger.LogContext, issuerInfo
 
 func (r *certReconciler) writeCertificateSecret(logctx logger.LogContext, issuerInfo utils.IssuerInfo, objectMeta metav1.ObjectMeta,
 	certificates *certificate.Resource, specHash string, specSecretRef *corev1.SecretReference, requestedAt *time.Time) (*corev1.SecretReference, error) {
-	secret := &corev1.Secret{}
+	secret := &corev1.Secret{
+		Type: corev1.SecretTypeTLS,
```

Comment:
If we set it here, do we still need the special handling in line 853 (migration case Cert-Manager)?

Reply:
Thanks, good point! I removed the obsolete code lines 844ff